manager/allocator/network.go

```diff
@@ -1106,6 +1109,7 @@ func (a *Allocator) procTasksNetwork(ctx context.Context, onRetry bool) {
 		quiet = true
 	}
 	allocatedTasks := make([]*api.Task, 0, len(toAllocate))
+	errorTasks := make([]*api.Task, 0, len(toAllocate))
```

Comment:
Where do tasks get added to this slice?

Reply:
Whoops, forgot to update this code block. Fixed.